hazelcast/src/test/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImplTest.java

```diff
@@ -35,7 +34,7 @@
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
-import javax.swing.JPanel;
+import javax.swing.*;
```

Comment:
I suppose checkstyle won't be happy with this?

Reply:
Oh! how did that happen? 😮